pages/management/SettingsHandler.php

```diff
@@ -53,6 +55,27 @@ public function __construct()
     public function workflow($args, $request)
     {
         parent::workflow($args, $request);
+
+        $templateMgr = TemplateManager::getManager($request);
+
+        $context = $request->getContext();
+        $dispatcher = $request->getDispatcher();
+
+        $contextApiUrl = $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), 'contexts/' . $context->getId());
+
+        $locales = $context->getSupportedFormLocaleNames();
+        $locales = array_map(fn (string $locale, string $name) => ['key' => $locale, 'label' => $name], array_keys($locales), $locales);
+
+        $reviewGuidanceForm = new \APP\components\forms\context\ReviewGuidanceForm($contextApiUrl, $locales, $context);
+        $reviewSetupForm = new PKPReviewSetupForm($contextApiUrl, $locales, $context);
+
+        $templateMgr->setState([
+            'components' => [
+                FORM_REVIEW_GUIDANCE => $reviewGuidanceForm->getConfig(),
+                FORM_REVIEW_SETUP => $reviewSetupForm->getConfig(),
+            ],
+        ]);
```

Comment:
Did you test this after making the change? I think that this will overwrite the `components` key of the state array, so all the existing components will be removed. I _think_ you need to try something like this:

```php
$components = $templateMgr->getState('components');
$components[FORM_REVIEW_GUIDANCE] = ...;
$components[FORM_REVIEW_SETUP] = ...;
$templateMgr->setState(['components' => $components]);
```

Reply:
@NateWr you are right. I changed it.